generator-scrivito/generators/obj/index.js

```diff
@@ -0,0 +1,98 @@
+const Generator = require("yeoman-generator");
+
+module.exports = class extends Generator {
+  constructor(args, opts) {
+    super(args, opts);
+    this.log("Welcome in scrivito Obj/Widgets generator!");
+  }
+
+  start() {
+    this._creatingObj();
+  }
+
+  _creatingName(name) {
+    const defaultName = name;
+    return defaultName;
```

Comment:
This method `_creatingName` does not provide any value what so ever. Please remove it.